packages/ai-bot/tests/prompt-construction-test.ts

```diff
@@ -29,6 +29,48 @@ function getPatchTool(cardId: string, properties: any) {
   };
 }
 
+function getSearchTool(cardId: string) {
```

Comment:
actually, all the context was passed in here. I never updated the system prompt at all. Not sure if its worth while to do so.